### ISSUE TITLE
Scroll Bar Problems in Theia IDE

### ISSUE THREAD

**Issue by tsmaeder** [eclipse-theia/theia#14586]
### Bug Description:
In Theia IDE, tree views like the Explorer cannot be scrolled: there is an overlay covering 100% coming in when hovering over the the vertical scroll bar.

### Steps to Reproduce:

1. Open the navigator on a Theia repo, expand node_modules
2. Observe: you get a scroll bar with a thumb of some length
3. Try to grab the scroll-thumb and move it
4. Observe: you can't.

### Additional Information

- Operating System: Windows 11
- Theia Version: 1.56.1 Theia IDE

Since we can't fix this for adopters who build their own Theia-based products, I've sent the following message to the dev mailing list for a work around:

> we've noticed a problem when creating the new IDE release 1.56.1: https://github.com/eclipse-theia/theia/issues/14586. We are, of course, preparing a new release. The problem was caused by picking up a new point version of the "perfect-scrollbar" dependency. The 1.5.6 version is not working with Theia and we're pinning the version to 1.5.5 in Theia IDE. Unfortunately, we cannot force clients of the @theia packages to use the 1.5.5 version due to limitations in yarn.
> 
> So if you build your own Theia-based product you should pin the version as well by adding the following to the package.json at the root of your monorepo:
> 
> "resolutions": {
>     "**/perfect-scrollbar": "1.5.5"
>  },
> 
> You can also check which version of perfect-scrollbar you're using in your product by doing "yarn why perfect-scrollbar". If the versions showing up are < 1.5.6, you're fine.
> 
> 
> Being unable to update this dependency is not desirable in the long run and we'll investigate what the root cause of the problem is and take appropriate action.
> 
> hope that helps
> 
> /Thomas
>

**Comment by tsmaeder** [comment]
![scroll_issue](https://github.com/user-attachments/assets/9c41c8a8-8f7b-4e29-87d5-fb5c1459891e)

**Comment by tsmaeder** [comment]
Note: you can't reproduce this in Theia master. Turns out Theia IDE picked up an update of `perfect-scrollbar` from 1.5.5 to 1.5.6. We need to pin `perfect-scrollbar` at 1.5.5 for the time being.

**Comment by tsmaeder** [comment]
Turns out the pinning does not work like this for adopters.

**Comment by sdirix** [comment]
What we *could* do is to add a patch to `theia-patch` which reverts the changes of perfect-scrollbar 1.5.6. `patch-package` is aware of versions and will only apply the patch on the broken version.

However I think that many adopters are not even using `theia-patch`, so that will not help them.

**Comment by tsmaeder** [comment]
> What we could do is to add a patch to theia-patch which reverts the changes of perfect-scrollbar 1.5.6.

I don't think that's a sustainable solution. When 1.5.7 comes out we'll have to update the patch and that ad infinitum?

**Comment by sdirix** [comment]
I agree, it's absolutely not a long term solution. The only "real" solutions are:
 - Report/Fix/Release a fix upstream, i.e. in `perfect-scrollbar`
 - Workaround/Fix the issue within Theia
 - Get rid of `perfect-scrollbar`

The suggestion of `theia-patch` would just be a convenience workaround for adopters until the issue is actually resolved. I'm definitely fine with not doing that.

**Comment by sgraband** [comment]
This happens again for me in Theia IDE preview 1.60.0 in the search view.

**Comment by tsmaeder** [comment]
@sgraband what version do we have in TheiaIDE?

**Comment by sgraband** [comment]
Theia version is this commit: https://github.com/eclipse-theia/theia/commit/9332521b9aa74bf534d45968905d37e30a13c3b5 (current master)

**Comment by sgraband** [comment]
Seems like this is no longer working: https://github.com/eclipse-theia/theia-ide/blob/0c8bf2bb74485e7ab5a9ca08257b59a7be28012a/yarn.lock#L12088-L12094

Which is weird because we also have the correct block:https://github.com/eclipse-theia/theia-ide/blob/0c8bf2bb74485e7ab5a9ca08257b59a7be28012a/yarn.lock#L11565C1-L11568C108

**Comment by tsmaeder** [comment]
What does `yarn why` say about the package?

**Comment by tsmaeder** [comment]
@sgraband  I can't reproduce this on Windows on 1.60.0 IDE. Do you have steps to reproduce?

**Comment by sgraband** [comment]
Using yarn why i noticed that we have `perfect-scrollbar` and `react-perfect-scrollbar`. So for `perfect-scrollbar` we still have the correct version.

I have no reproducer it, but in the search view it just seems to be there all the time 🤷

**Comment by tsmaeder** [comment]
@sgraband I'm closing this issue as it's clearly not the same issue (perfect-scrollbar is the correct version). @sgraband I still don't see this problem, even in the search view. Could you open a new issue describing the problem in some detail, please?
